examples/Utils/b3Clock.cpp

```diff
@@ -143,8 +141,41 @@ unsigned long int b3Clock::getTimeMilliseconds()
 #endif
 }
 
-	/// Returns the time in us since the last call to reset or since 
-	/// the Clock was created.
+/// Gets the system time in milliseconds
+unsigned long int b3Clock::getSystemTimeMilliseconds() {
```

Comment:
No it is not a copy. I is not a difference to the last time (currentTime.QuadPart - m_data->mStartTime.QuadPart) but it is the absolute system time. I use this to get a seed for randomness in the evolutionary algorithm. Do you propose another way to get a random seed?